go/cmd/vtctldclient/command/transactions.go

```diff
@@ -0,0 +1,134 @@
+/*
+Copyright 2024 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package command
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"vitess.io/vitess/go/cmd/vtctldclient/cli"
+	querypb "vitess.io/vitess/go/vt/proto/query"
+	vtctldatapb "vitess.io/vitess/go/vt/proto/vtctldata"
+)
+
+var (
+	DistributedTransaction = &cobra.Command{
+		Use:   "DistributedTransaction <cmd>",
+		Short: "Perform commands on distributed transaction",
+		Args:  cobra.MinimumNArgs(2),
+
+		DisableFlagsInUseLine: true,
+	}
+
+	// GetUnresolvedTransactions makes an GetUnresolvedTransactions gRPC call to a vtctld.
+	GetUnresolvedTransactions = &cobra.Command{
+		Use:     "list <keyspace>",
+		Short:   "Retrieves unresolved transactions for the given keyspace.",
+		Aliases: []string{"List"},
+		Args:    cobra.ExactArgs(1),
+		RunE:    commandGetUnresolvedTransactions,
+
+		DisableFlagsInUseLine: true,
+	}
+
+	// ConcludeTransaction makes a ConcludeTransaction gRPC call to a vtctld.
+	ConcludeTransaction = &cobra.Command{
+		Use:     "conclude <dtid> [<keyspace/shard> ...]",
```

Comment:
Should we even take in <keyspace/shard> list? Why don't we just conclude the transaction everywhere. Like the user only has one option, which is to conclude the transaction on all the shards, not on some shards.

Reply:
I am thinking about it that if the list of keyspace/shard is not provided we will get it and then conclude on all.
This is more from if the user gets the unresolved transactions list then it knows which all shards are involved and can provide the list to conclude.

We can make the flow better. Let's do that in a follow up PR.